include/carma_bits/typecheck.h

```diff
@@ -34,7 +34,10 @@ using is_Vec = std::enable_if_t<p_is_Vec<armaT>::value, int>;
 template <typename armaT>
 using is_Mat = std::enable_if_t<arma::is_Mat<armaT>::value, int>;
 template <typename armaT>
-using is_Mat_only = std::enable_if_t<arma::is_Mat_only<armaT>::value, int>;
+using is_Mat_only = std::enable_if_t< arma::is_Mat_only<armaT>::value &&
```

Comment:
@yaqwsx `arma::is_Mat_only<>` is also internal Armadillo functionality, subject to change or removal without notice.

Reply:
Noted